src/main/java/utils/XssFilter.java

```diff
@@ -156,7 +156,7 @@ public static String levelThree (String input)
 		input = input.replaceAll("script", "scr.pt");
 		for(int h = 0; h < FindXSS.javascriptTriggers.length; h++)
 		{
-			for(int i = 0; i <= 3; i++)
+			for(int i = 0; i <= 1; i++)

```

Comment:
Looks like this change was unintended. Can it be reverted?

Reply:
See #527. I strongly advocate for keeping this change as I've seen many students (at this point we have had >300 people use shepherd) struggle with this specific filter.